pysts/loader.py

```diff
@@ -172,6 +173,35 @@ def load_sts(dsfile, skip_unlabeled=True):
             s1.append(word_tokenize(s1x))
     return (s0, s1, np.array(labels))
 
+def load_snli(dsfile, vocab):
+    s0i = []
+    s1i = []
+    labels = []
+    lmappings={'contradiction': np.array([1,0,0]), 'neutral':np.array([0,1,0]) , 'entailment': np.array([0,0,1])}
+    i = 0
+    skips=0
+    neutral_skips=0
+    with open(dsfile) as f:
+        for l in f:
+            d=json.loads(l)
+            if i % 5000 == 0:
+                print('%d samples read, %d no label skips, %d neutral label skips' % (i,skips, neutral_skips))
+            if len(d['gold_label'])<2: # some pairs are not labeled, skip them
+                skips += 1
+                continue
+            label = d['gold_label']
+            if label in lmappings:
+                s0 = word_tokenize(d['sentence1'])
+                s1 = word_tokenize(d['sentence2'])
+                s0i.append(s0)
+                s1i.append(s1)
+                labels.append(lmappings[label])
+            else:
+                neutral_skips+=1
```

Comment:
Yes it does happen. After the authors collected the dataset, they performed an additional round of data validation on a subsample. If 3 of 5 annotators agreed on one label, the sentence pair was assigned that label. If there was no such consensus, the pair was given a label ''-".

Will add that to comment in code.